includes/multi-currency/client/settings/multi-currency/enabled-currencies-list/index.js

```diff
@@ -100,7 +83,6 @@ const EnabledCurrencies = () => {
 								<EnabledCurrenciesListItem
 									key={ enabledCurrencies[ code ].id }
 									currency={ enabledCurrencies[ code ] }
-									defaultCurrency={ defaultCurrency }
```

Comment:
`defaultCurrency` can be gathered by the `EnabledCurrenciesListItem` component without prop-drilling